aws/resource_aws_db_parameter_group.go

```diff
@@ -149,7 +149,6 @@ func resourceAwsDbParameterGroupRead(d *schema.ResourceData, meta interface{}) e
 	// Only include user customized parameters as there's hundreds of system/default ones
 	describeParametersOpts := rds.DescribeDBParametersInput{
 		DBParameterGroupName: aws.String(d.Id()),
-		Source:               aws.String("user"),
```

Comment:
If it only was so simple. If you do that, then the `read` will show a giant list of parameters, including ones you never set. What you need is filter those to only include the ones you have in your state.